main.tf

```
@@ -53,11 +61,11 @@ resource "aws_security_group" "default" {
 
 module "dns" {
   source    = "git::https://github.com/cloudposse/terraform-aws-route53-cluster-hostname.git?ref=tags/0.2.5"
-  enabled   = "${length(var.zone_id) > 0 ? "true" : "false"}"
+  enabled   = "${var.enabled}"
```

```suggestion
  enabled   = "${local.enabled && length(var.zone_id) > 0 ? "true" : "false"}"
```

We do it in many modules.
Don't create the DNS record if zone ID is not provided